omod/src/main/java/org/openmrs/module/appframework/web1x/AppFrameworkHomepagePortletController.java

```diff
@@ -1,100 +0,0 @@
-/**
- * The contents of this file are subject to the OpenMRS Public License
- * Version 1.0 (the "License"); you may not use this file except in
- * compliance with the License. You may obtain a copy of the License at
- * http://license.openmrs.org
- *
- * Software distributed under the License is distributed on an "AS IS"
- * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
- * License for the specific language governing rights and limitations
- * under the License.
- *
- * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
- */
-package org.openmrs.module.appframework.web1x;
-
-import java.io.IOException;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import javax.servlet.ServletException;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
-import org.openmrs.User;
-import org.openmrs.api.context.Context;
-import org.openmrs.module.appframework.service.AppFrameworkService;
-import org.openmrs.module.appframework.domain.AppDescriptor;
-import org.openmrs.util.PrivilegeConstants;
-import org.openmrs.web.controller.PortletController;
-import org.springframework.web.servlet.ModelAndView;
-
-/**
- *
- */
-public class AppFrameworkHomepagePortletController extends PortletController {
```

Comment:
I said to get rid of this, actually. We can drop support for the legacy UI within this module with this upcoming release, because nobody is actually using that feature.